lms/static/js/student_account/components/spec/PasswordResetConfirmation_spec.js

```diff
@@ -0,0 +1,68 @@
+/* globals setFixtures */
+
+import ReactDOM from 'react-dom';
+import React from 'react';
+import sinon from 'sinon'; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
I know that adding *another* testing framework is what we all want, but have you thought about [`jest`](https://github.com/facebook/jest/) and/or [`enzyme`](https://github.com/airbnb/enzyme)?

Reply:
I did have some difficulty making everything work with React. I dunno if those fix it for me. Do they make working with React async's setState any easier? Jasmine and async is a big pain in the butt. I just barely managed to get it to work with an async fetch(), but multiple sequential async setStates broke my brain and I avoided it.

Reply:
Though I'm not necessarily excited about refactoring this test file now that I've got it.